shell/pages/support/index.vue

```diff
@@ -93,11 +93,17 @@ export default {
     },
 
     supportConfigLink() {
-      if (!this.cspAdapter) {
+      const adapter = this.cspAdapter;
+
+      if (!adapter) {
         return false;
       }
 
-      return `${ this.serverUrl }/v1/generateSUSERancherSupportConfig`;
+      if (adapter.metadata.name === 'rancher-csp-billing-adapter') {
+        return `${ this.serverUrl }/v1/generateSUSERancherSupportConfig?usePAYG`;
```

Comment:
is this intentionally not setting a value for the parameter? is the backend just checking for the presence of the parameter rather than its value?

Reply:
I see in the corresponding backend change ( https://github.com/rancher/rancher/pull/41849/files#diff-5d7133457c7014b056d00275cc680aa1b50aa47b72b241be15dc4269b359496bR65 )  you're checking for the presence of the parameter, so that matches up... if the dashboard team is ok with that it works for me :)

Reply:
I had a similar comment on the backend side - `usePAYG=true` would read more cleanly to me, but I'm ok with it if the UI is (and we can find an RFC clarifying that single value parameters like this are ok).

Reply:
We're fine UI side. We don't do anything other than stick it in an anchor.  

The URI / HTML specs are a bit vague/absent about a missing value -https://www.rfc-editor.org/rfc/rfc3986#section-3.4.

Generally i think it's avoided. If there's no clear spec it probably gets treated differently by different backends (empty string/null/omitted/etc). If the place this is used handles it ok, i'd say we're good.

Reply:
I've updated the backend https://github.com/rancher/rancher/pull/41849 to explicitly look for `usePAYG=true` as well.